src/main/java/com/google/api/codegen/transformer/php/PhpModelTypeNameConverter.java

```diff
@@ -262,7 +262,7 @@ public String renderPrimitiveValue(TypeRef type, String value) {
         return value.toLowerCase();
       case TYPE_STRING:
       case TYPE_BYTES:
-        return "\"" + value + "\"";
```

Comment:
Please change to `'\''` here and in all other places (except `PRIMITIVE_ZERO_VALUE` initialization, since that one actually expects `String`). It is not a big deal, but `'\''` is slightly less heavy, since it is just a `char` not a `String`.